packages/block-library/src/columns/style.scss

```diff
@@ -56,7 +56,7 @@
 
 		// Columns with an explicitly-assigned width should maintain their
 		// `flex-basis` width and not grow.
-		&[style] {
+		&[style*="flex-basis"] {
```

Comment:
Are we now applying other styles? This was already pretty fragile in the first place, but was a compromise since existing content wouldn't have any indicators of whether a predefined width was assigned. I think if we expand this, we might want to consider deprecations to include some more durable indicator (like a `data-` attribute, or class name).

Reply:
Yes, there are other inline styles for block animations. Yes, would probably be best to have a separate attribute.